pkg/openai/openai.go

```diff
@@ -149,11 +151,14 @@ func (openAI *OpenAIClient) GenerateCode(input string) (string, error) {
 		return "", err
 	}
 
-	// get the first response
-	response, err := completionResponse.GetFirstChoice()
+	// get n responses
+	response, err := completionResponse.GetNChoices(openAI.NCompletions)
```

Comment:
just return these as a list, don't concatenate

Reply:
Updated, see my comment to @guymguym's comment above.